src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentPrice = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+  const totalCost = days * rentPrice;
+  const threeDays = 3;
+  const sevenDays = 7;
```

Comment:
```suggestion
  const daysForSmallDiscount = 3;
  const daysForBigDiscount = 7;
```
![image](https://user-images.githubusercontent.com/96409563/212984416-a8c759de-b1eb-4d17-baf2-3a0ae192e335.png)